### DISCUSSION TITLE
Deploying contract SimpleStorage

### DISCUSSION THREAD

**Discussion by Nobleraj** [Cyfrin/foundry-full-course-cu#2233] [General]
Have created the zkSync sepolia wallet and deployed the SimpleStorage contract using the zkSync account in metamask in remix. 

For a while i'm seeing the message like 'Deploying contract SimpleStorage'

It would take more time to deploy the contract in zkSync sepolia testnet ?

<img width="1135" alt="Screenshot 2024-08-13 at 3 18 30 PM" src="https://github.com/user-attachments/assets/268829e0-7924-49a6-9815-a2e2379bfe4a">

**Comment by EngrPips** [comment]
Hello @Nobleraj, It seems like working with `ZkSync` on `Remix` has been recently slow. It might take longer than usual or not work as seamlessly as expected.

**Reply by Nobleraj** [reply]
@EngrPips thankyou for the prompt response. I really appreciate it. 

let me wait for sometime and check.

**Reply by EngrPips** [reply]
You can also refresh your browser and retry.

**Reply by Nobleraj** [reply]
sure, let me try

**Reply by EngrPips** [reply]
👍

**Reply by Nobleraj** [reply]
it works @EngrPips  thankyou 🚀

**Reply by EngrPips** [reply]
Keep crushing it, Friend.
